Pipfile

```diff
@@ -9,15 +9,15 @@ rope = "*"
 pep8 = "*"
 
 [packages]
-flask = "*"
-requests = "*"
-pyproj = "*"
-geojson = "*"
-flask-compress = "~=1.4.0"
-flask-cors = "*"
-gunicorn = "*"
-flasgger = "*"
-python-dotenv = "~=0.10.3"
+flask = "==1.1.1"
```

Comment:
The use of `~=` is preferred over the `==` identifier as the latter prevents pipenv from updating the packages

[Source](https://docs.pipenv.org/en/latest/basics/#specifying-versions-of-a-package)